css/properties/break-inside.json

```diff
@@ -82,7 +82,7 @@
               "opera": "mirror",
               "opera_android": "mirror",
               "safari": {
-                "version_added": "≤13.1"
+                "version_added": "10"
```

Comment:
According to https://github.com/caugner/css-properties/commit/aeba5c2cbf6970c1e9f7fead1d5afaa7073e1597, the unprefixed property was only added in Safari 13.1, but it's plausible that Safari 10 introduced `break-inside` as an alias for `-webkit-column-break-inside`, which was [added in Safari 3](https://github.com/caugner/css-properties/commit/c1ecbe388db1bf60297f5a08d23e6d4471f4fc85#diff-b63d72471087ee495d4f5c7e94b58bef200825135bdee1bdcddbfbd594b0c328R27), which was [removed in Safari 16.5](https://github.com/caugner/css-properties/commit/9b4cad162976314a2a1bc7364e904f1f21b0d1c3#diff-b63d72471087ee495d4f5c7e94b58bef200825135bdee1bdcddbfbd594b0c328L28).